src/aquto.js

```diff
@@ -89,22 +80,8 @@ function genericCheckEligibility(options) {
  */
 function checkOfferWallEligibility(options) {
   var data = { apiVersion: 'v8' }
-  if(options.carrier) {
-    data.operatorCode = options.carrier
-  }
-  if(options.phoneNumber) {
-    data.phoneNumber = options.phoneNumber
-  }
-  if(options.countryCode) {
-    data.countryCode = options.countryCode
-  }
```

Comment:
Make sure any field that's deleted here gets included e.g. countryCode and carrier (since those fields are only for this method it probably makes sense to just leave them here and not include in the generic apply params